multiqc/multiqc.py

```diff
@@ -746,6 +753,28 @@ def __rich_measure__(self, console: rich.console.Console, options: rich.console.
         report.runtimes["mods"][run_module_names[mod_idx]] = time.time() - mod_starttime
     report.runtimes["total_mods"] = time.time() - total_mods_starttime
 
+    # Parse software version from config if provided
+    versions_from_config = software_versions.load_versions_from_config(config)
+    for software, versions in versions_from_config.items():
+        # Try to find if the software is listed among the executed modules.
+        # Unlisted software are still reported in the `Software Versions` section.
+        module = software_versions.find_matching_module(software, report.modules_output)
+        if module is None:
+            logger.debug(f"No executed modules matches '{software}' listed in config")
+        else:
```

Comment:
Repeat of https://github.com/ewels/MultiQC/pull/1927#discussion_r1309398815 as it became outdated.

I don't think that this debug log message is particularly helpful. I think it can be removed.

```suggestion
        if module is not None:
```

Reply:
This debug message is now removed